react-js/styling-react.md

```diff
@@ -0,0 +1,238 @@
+# Styling in React
+
+### Projected Time : 1 hr
+- Lesson: 40 mins.
+- Guided Practice: 20 mins.
+
+### Prerequisites
+- [CSS Lesson](/css/css)
+- [Bootstrap Lesson](/bootstrap/bootstrap)
+- [React Lesson](react)
+
+
+### Motivation
+
+Styling has been one of the building blocks of web development. Cascading Style Sheets is a style sheet language used for describing the presentation of a document written in a markup language like HTML. React only provides a virtual DOM, whereas the presentation part still lies with CSS.
+
+### Objectives
+
+- Styling in JS Framework.
+- Inline styling and using external Stylesheet.
+- Bootstrap in React.	 
+
+### Specific Things To Learn
+
+- Defining styles as objects.
+- Importing stylesheets into react application.
+
+### Materials
+
+- [Video: Styling of React App in 3 different ways](https://www.youtube.com/watch?v=j5P9FHiBVNo) - Using external CSS, internal and CSS modules.
+- [Video: Bootstrap in React](https://www.youtube.com/watch?v=wdbzLi0tWq0) - Bootstrap in React can be used using CDN or by installing the package. 
+
+### Lesson
+
+- [Slides: CSS in React ](https://drive.google.com/open?id=15oauTP4afibqMvNKW5XgVC1qraYUKxY-)
+
+### Guided Practice
+
+> #### Let's create a simple div of flex box using InLine CSS then we will achieve same using External CSS and Bootstrap separately.
+
+- First thing we need to do is create a react application.
+	- Create a react app using create react app command. `npx create-react-app`.
+    - This will create a simple react app , and now we will start making changes to this app.
+	- First of all get rid of all the unnecessary stuff showing on DOM , remove all the lines in render of `App.js`. 
+	- Now we create a simple div **parent div** which will contain few div as *child* elements.
+	- After creating that , your ``App.js`` will look like this 
+	- ```
+		import React from 'react';
```

Comment:
This is great!  Please fix the indentation in the example.

Reply:
Okay sure.